src/admin/pages/users/users-store.js

```diff
@@ -0,0 +1,185 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+
+    state: {
+        users: [
```

Comment:
Move this data to json file in data folder and import here.

Reply:
Ok Sure